src/core/psxmem.h

```diff
@@ -45,10 +45,10 @@ namespace PCSX {
 
 class Memory {
   public:
-    uint8_t *m_psxM = NULL;  // Kernel & User Memory (8 Meg)
-    uint8_t *m_psxP = NULL;  // Parallel Port (64K)
-    uint8_t *m_psxR = NULL;  // BIOS ROM (512K)
-    uint8_t *m_psxH = NULL;  // Scratch Pad (1K) & Hardware Registers (8K)
+    uint8_t *m_main = NULL;        // Kernel & User Memory (8 Meg)
+    uint8_t *m_exp1 = NULL;        // Expansion Region 1 (ROM/RAM) / Parallel Port (512K)
+    uint8_t *m_bios = NULL;        // BIOS ROM (512K)
+    uint8_t *m_scratchpad = NULL;  // Scratch Pad (1K) & Hardware Registers (8K)
```

Comment:
Renaming this to "scratchpad" hides the fact this also contains a copy of hardware registers. The H used to stand for Hardware. Could use `m_hard` to keep alignment?

Reply:
m_wram over `m_main` probably makes more sense too. Also miiiight want to squash some of these commits.